packages/deploy/deploy/700_land/land/1000_deploy_land_registry.ts

```diff
@@ -0,0 +1,31 @@
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
i think the idea was to keep the same numbers 
folder: 700
> scripts: 701, 702, etc

Reply:
Why? It limits the amount of numbers you can use.

Reply:
It is even better if we stop using numbers in directories (usually we don't have an order there is just tag dependencies).